### PR TITLE
Update draft-secure-credential-transfer.md

### PR DESCRIPTION
**Pull request by dimmyvi**
Addressing review comments

### CODE REVIEW

**Comment by dimmyvi** [comment]
Added RelinquishMailbox endpoint - to unbind invalid Receiver device

File: draft-secure-credential-transfer.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ API parameters:
 
 # Credential transfer workflows
 
-We define two flows for credential transfer: 1. Stateless (Relay server facilitates a single credential data transfer: Sender -> Relay -> Receiver) and 2. Stateful (Relay server facilitates additional data transfers - there are multiple data transfers in this flow to prepare credential data for registering or provisioning by Receiver). The details are provided below.
+We define two flows for credential transfer: 1. Stateless (Relay server facilitates a single credential data transfer: Sender -> Relay -> Receiver) and 2. Stateful (Relay server facilitates additional data transfers - there are multiple data transfers in this flow to prepare credential data for registering or provisioning by Receiver). Relay server does not limit the number of such data tranfsfers between Sender and Receiver devices. The details are provided below.
```
Comment (alexpell00) [inline draft-secure-credential-transfer.md:117]: While the Relay server doesn't limit the number of round trips we've said that reusing the same symmetric key isn't sufficient from a security POV. I don't know if we need to include that here, but wanted to call out that if we were to use this for a chatty provisioning we might need to use the first few messages to upgrade the channel's encryption.

File: draft-secure-credential-transfer.md [hunk 2]
```diff
@@ -538,6 +538,42 @@ Unauthorized - calling device is not authorized to read the secure content of th
...
+### Request Parameters
+
+Path parameters:
+
+- version (String, Required) - the version of the API. At the time of writing this document, “v1”.
+- mailboxIdentifier(String, Required) - MailboxIdentifier (refer to Terminology).
+
+Header parameters:
+
+- deviceAttestation (String, Optional) - optional remote OEM device proprietary attestation data.
```
Comment (alexpell00) [inline draft-secure-credential-transfer.md:558]: If this is optional then it doesn't think it provides any security benefit. A bad actor could just not send one up. @matthewcbyington what do you think?

**Comment by dimmyvi** [comment]
Merging into main branch

**Pull request merged**